backend/nodes/utils/color_transfer.py

```diff
@@ -86,7 +86,6 @@ def color_transfer(
         ref_img = cv2.cvtColor(ref_img, cv2.COLOR_BGR2LAB)
     elif colorspace == "RGB":
         a_clip_min, a_clip_max = (0, 1)
-        b_clip_min, b_clip_max = (0, 1)
```

Comment:
Why did you get rid of this line?

Reply:
looooool It got caught accidentally while deleting Luv